src/lib/auth.config.ts

```diff
@@ -17,11 +21,16 @@ export default {
         if (validatedFields.success) {
           const { email, password } = validatedFields.data;
 
-          const user = await prisma.landlord.findUnique({
-            where: { email },
-          });
+          const user = await getLandlordByEmail(email);
 
-          if (!user) return null;
+          // If the user does not exist or the email or password is not provided
+          if (!user || !user.email || !user.password) return null;
+
+          // If the user has not verified their email, throw an error
+          if (!user.emailVerified) {
+            await sendVerificationEmail(user.email);
```

Comment:
Could you add a delay to the verification emails? Right now, an email is sent every time someone tries to log in. Since the API has daily limits, it would be better to send an email only once every 30 seconds, or something similar.

Reply:
I added a delay to resend the verification emails. We don't resend an email 30 minutes after the email is sent.